pkg/rc/replication_controller_test.go

```diff
@@ -185,7 +186,7 @@ func TestSchedule(t *testing.T) {
 	rc.WatchDesires(quit)
 
 	rcStore.SetDesiredReplicas(rc.ID(), 1)
-	numNodes := waitForNodes(t, rc, 1)
+	numNodes := waitForNodes(t, rc, 7)
```

Comment:
should this really have changed? waits for 7 (I assume that times out), then asserts 1